Client/Frontend/Browser/Browser.swift

```diff
@@ -45,7 +45,9 @@ class Browser: NSObject, WKScriptMessageHandler {
 
     var title: String? {
         if let title = webView.title {
-        	return title
+            if !title.isEmpty {
+                return title
```

Comment:
Unrelated stuff? Also, is an empty title not valid? What do we do with them on Android? I always waffle back and forth on this, but probably title should always return the title and "displayTitle" or something should be used if we want to return the url...